host-configs/lc/toss_4_x86_64_ib_cray/amdclang-xnack.cmake

```diff
@@ -0,0 +1,26 @@
+##############################################################################
+# Copyright (c) 2024, Lawrence Livermore National Security, LLC and CHAI
```

Comment:
Can you update the copyright years to 2016-25?

Reply:
Done